_posts/2017-11-30-totw-130.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Tip of the Week #130: Namespace Naming"
+layout: tips
+sidenav: side-nav-tips.html
+published: false
+permalink: tips/130
+type: markdown
+order: "130"
+---
+
+Originally posted as totw/130 on 2017-02-17
+
+By Titus Winters (titus@google.com)
+
+*The precision of naming takes away from the uniqueness of seeing* — Pierre
+Bonnard
+
+The earliest commit of the Google C++ Style Guide contains the guidance that
+many people are still using for namespace naming. Roughly, this can be
+summarized as "namespaces are derived from package paths," although there's an
+interesting less-followed option that an entire top-level package
+(search/...) could choose a single flat top-level namespace. Following on
```

Comment:
Omit search/ or genericize ?

In either case, enclose in \` \`

Reply:
just dropping it, it's less relevant externally